src/components/mainLayout/aboutUsCard/aboutUsCards.module.scss

```diff
@@ -0,0 +1,97 @@
+.aboutUsCard {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	position: relative;
+	width: 171px;
```

Comment:
try to avoid odd values here and in other places even if they are described in the figma layout

Reply:
I need to change this values to 170px?

Reply:
You can set either 170 or 172. The main thing is that they are not odd, since pixels are counted differently on different platforms.